codacy/values.yaml

```diff
@@ -20,18 +21,19 @@ global:
     rabbitmqUsername: rabbitmq
     rabbitmqPassword: rabbitmq
     rabbitmqvHost: "/"
-    host: codacy-rabbitmq-ha.codacy.svc.cluster.local
+    host: codacy-rabbitmq-ha
     port: 5672
     ssl: false
     timeout: 15s
+    fullnameOverride: codacy-rabbitmq-ha
```

Comment:
I don't think this will work.

https://github.com/helm/charts/blob/master/stable/rabbitmq-ha/templates/_helpers.tpl#L14

Reply:
This will be in `global.rabbitmq.fullnameOverride`. The rabbitmq-ha chart dosen't read values from here for this.